### ISSUE TITLE
/dominoscoupons Should Use FrugalFeeds

### ISSUE THREAD

**Issue by 49Indium** [UQComputingSociety/uqcsbot-discord#204]
[As brought up by lucid8426 on discord](https://discord.com/channels/813324385179271168/836243768411160606/1256604024501964955), the `/dominoscoupons` command provides discounts that may not work. Potentially we should also include [FrigralFeeds](https://www.frugalfeeds.com.au/) coupons.

One way to do this would be to add an optional argument for the source. I'm unsure what should happen by default: either just selecting one of the sources or trying to combine them. Ideally we would have some way to test if the coupons work, but I'm unsure if that is achievable and isn't required.

Note that we should probably still keep `https://www.couponese.com/store/dominos.com.au/` in some form, as a good back-up as some of the coupons still work.

**Comment by tj-dodd** [comment]
I've made decent progress on this issue. The user is now able to specify whether they want a list of codes from FrugalFeeds or Couponese through an optional argument, and by default the user is provided with a combined list. Right now the combined list is a random selection of discount codes (as you end up with a pretty big list), and I'm thinking of implementing a button the user can press to refresh the combined list - although what I've got might work well enough as is.
